docs/modules/ROOT/pages/references/quality-requirements/reliability/portal-api-data-recoverability.adoc

```diff
@@ -0,0 +1,26 @@
+= Data compatibility with external components
+
+Source::
+{product} administrator
+
+Stimulus::
+Reinstall APPUiO Global from scratch
+
+Environment::
+{zone}
+
+Artifact::
+Portal API, external third-party component
+
+Response::
+Entities that have their lifecycle outside of {product} have their availability restored
+
+Response measure::
+Following entities are recovered at latest after 48 hours since reinstallation:
++
+* Invoices
```

Comment:
```suggestion
* Invoices
* List of APPUiO Zones
* Organizations
* Teams
* Users
```

I really don't like to have this list here as I'm pretty sure that this list falls behind very fast. I'd rather find a generic description of this list, like "APPUiO Cloud Core Entities".

"The {product} core entities are recovered at latest after X hours since reinstallation"

We can then have another page describing the core entities which we can amend as the architecture grows without having to change a requirement.

Reply:
I think we may have a misunderstanding. Organizations, teams and users are stored in Keycloak and thus are not external data. To my understanding, this requirement doesn't need to cover those. Otherwise you'd be addressing disaster recovery processes, right? We haven't talked about that yet.
This requirement is about restoring data and processes where APPUiO Cloud has no data authority thereof.